tests/test_html.py

```diff
@@ -0,0 +1,437 @@
+from textwrap import dedent
+
+import jinja2
+import pytest
+from jinja2 import Markup
+
+from dmcontent.content_loader import ContentManifest
+from dmcontent.html import to_html, to_summary_list_rows
+
+
+@pytest.fixture
+def content_summary():
+    content = ContentManifest(
+        [
+            {
+                "slug": "first_section",
+                "name": "First Section",
+                "questions": [
+                    {
+                        "id": "boolean_true",
```

Comment:
Are the `id`s in a real content manifest going to be snake case or camel case? It's a small thing really, but when you're scanning down a piece of code, casing helps differentiate between the python variables, the type strings that are always going to be the same, and test data that will vary from manifest to manifest. 

I'd probably go for something like `{"id": "myBooleanQ1", ... ` instead?

Reply:
camelCaseConfirmed - did some renaming.